docs/resources/compute_interface_attach_v2.md

```diff
@@ -6,130 +6,106 @@ page_title: "flexibleengine_compute_interface_attach_v2"
 
 ## Example Usage
 
-### Basic Attachment
+### Attach a port (under the specified network) to the ECS instance and generate a random IP address
 
 ```hcl
-resource "flexibleengine_vpc_v1" "example_vpc" {
-  name = "example-vpc"
-  cidr = "192.168.0.0/16"
-}
-
-resource "flexibleengine_vpc_subnet_v1" "example_subnet" {
-  name       = "example-vpc-subnet"
-  cidr       = "192.168.0.0/24"
-  gateway_ip = "192.168.0.1"
-  vpc_id     = flexibleengine_vpc_v1.example_vpc.id
-}
-
-resource "flexibleengine_compute_instance_v2" "example_instance" {
-  name            = "example-instance"
-  security_groups = ["default"]
-}
-
-resource "flexibleengine_compute_interface_attach_v2" "example_interface_attach" {
-  instance_id = flexibleengine_compute_instance_v2.example_instance.id
-  network_id  = flexibleengine_vpc_subnet_v1.example_subnet.id
-}
-
-```
+variable "instance_id" {}
+variable "network_id" {}
 
-### Attachment Specifying a Fixed IP
-
-```hcl
-resource "flexibleengine_compute_interface_attach_v2" "example_interface_attach" {
-  instance_id = flexibleengine_compute_instance_v2.example_instance.id
-  network_id  = flexibleengine_vpc_subnet_v1.example_subnet.id
-  fixed_ip    = "10.0.10.10"
+resource "flexibleengine_compute_interface_attach_v2" "test" {
+  instance_id = var.instance_id
+  network_id  = var.network_id
 }
-
 ```
 
-### Attachment Using an Existing Port
+### Attach a port (under the specified network) to the ECS instance and use the custom security groups
 
 ```hcl
-resource "flexibleengine_networking_port_v2" "example_port" {
-  name           = "port_1"
-  network_id     = flexibleengine_vpc_subnet_v1.example_subnet.id
-  admin_state_up = "true"
+variable "instance_id" {}
+variable "network_id" {}
+variable "security_group_ids" {
+  type = list(string)
 }
 
-resource "flexibleengine_compute_interface_attach_v2" "example_interface_attach" {
-  instance_id = flexibleengine_compute_instance_v2.example_instance.id
-  port_id     = flexibleengine_networking_port_v2.example_port.id
+resource "flexibleengine_compute_interface_attach_v2" "test" {
+  instance_id        = var.instance_id
+  network_id         = var.network_id
+  fixed_ip           = "192.168.10.199"
+  security_group_ids = var.security_group_ids
 }
 
 ```
 
-### Attaching Multiple Interfaces
+### Attach a custom port to the ECS instance
 
 ```hcl
-resource "flexibleengine_networking_port_v2" "example_ports" {
-  count          = 2
-  name           = format("port-%02d", count.index + 1)
-  network_id     = flexibleengine_vpc_subnet_v1.example_subnet.id
-  admin_state_up = "true"
-}
+variable "security_group_id" {}
 
-resource "flexibleengine_compute_interface_attach_v2" "example_attachments" {
-  count          = 2
-  instance_id = flexibleengine_compute_instance_v2.example_instance.id
-  port_id     = flexibleengine_networking_port_v2.example_ports.*.id[count.index]
+data "flexibleengine_vpc_subnet_v1" "mynet" {
+  name = "subnet-default"
```

Comment:
flexibleengine seems have no default vpc and subnet, it's better to use resource

Reply:
done